src/Mvc/Directory.Build.props

```diff
@@ -17,6 +17,7 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <UseRoutingFromFeatureBranch>true</UseRoutingFromFeatureBranch>
     <SharedSourceRoot>$(MSBuildThisFileDirectory)..\Shared\</SharedSourceRoot>
+    <NoWarn>$(NoWarn);NU1605</NoWarn>
```

Comment:
What package are we downgrading? And why?

Reply:
The deps of 18619-0194 are being downgraded to 18606-0099

Reply:
If the build works when you do the equivalent of `s/18606-0099/18619-0194/g`, do that. Don't worry about the other versioning schemes unless they cause NU1605.

If additional NU1605 warnings come up, do what they say to make this build.